Source/gmenu.cpp

```diff
@@ -39,7 +39,7 @@ void gmenu_draw_pause(const CelOutputBuffer &out)
 		RedBack(out);
 	if (sgpCurrentMenu == nullptr) {
 		light_table_index = 0;
-		DrawString(out, _("Pause"), { PANEL_LEFT + 252, 176, 0, 0 }, UIS_MED, 2);
+		DrawString(out, _("Pause"), { PANEL_LEFT + 252, PANEL_TOP / 2, 0, 0 }, UIS_HUGE, 2);

```

Comment:
I wonder when this regressed... it must've been a long time ago right?

Reply:
looks like it has always been like this, only centered horizontally but not vertically, I think when people saw it near top of the screen on different resolutions, they assumed it was a feature